clients/broadcaster.go

```diff
@@ -61,19 +61,16 @@ type EncodedProfile struct {
 	ChromaFormat int32  `json:"chromaFormat,omitempty"`
 }
 
+var client = &http.Client{
```

Comment:
Did some testing and didn't see any negative performance impact from reusing the HTTP client and Transport.